site/src/pages/404.js

```diff
@@ -2,25 +2,48 @@ import React from 'react';
 import { graphql } from 'gatsby';
 
 import Layout from '../components/layout';
+import PageHead from '../components/head';
+import SiteMap from '../components/site-map';
+
+const NotFoundPage = ({ data, location }) => {
+  const nodes = data.allMdx.nodes;
 
-const NotFoundPage = ({ location }) => {
   return (
     <Layout location={location}>
-      <h1>Not Found</h1>
-      <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
+      <div className="content">
+        <h2>404 Not Found</h2>
+        <p>
+          Sorry, the url you're looking for appears to be missing. Check out the
+          sitemap below to find your expected page.
+        </p>
+        <SiteMap location={location} nodes={nodes} />
```

Comment:
With the url updates, I went ahead and enhanced the 404 page with a sitemap so a user could hopefully find what they were looking for.

Reply:
I liked the sadness

Reply:
Sadness is always available.